doc/admin-guide/files/ssl_multicert.config.en.rst

```diff
@@ -106,8 +106,9 @@ ssl_ticket_enabled=1|0 (optional)
   OpenSSL should be upgraded to version 0.9.8f or higher. This
   option must be set to `0` to disable session ticket support.
 
-ticket_key_name=FILENAME (optional) [**REMOVED in 7.1.x and 8.0**]
-   Ticket key should be set in records.config via :ts:cv:`proxy.config.ssl.server.ticket_key.filename`
+ssl_ticket_number=INTEGER (optional)
+  Specifies the number of TLSv1.3 session tickets that are issued.
```

Comment:
I think TLS v1.2 also supports ssl ticket.
Is this config only for TLS v1.3? If yes, the change looks good.

Reply:
The linked manpages from the original commit say it's a 1.3 feature.

https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_num_tickets.html

Reply:
Looks good. The ssl ticket itself is in TLS 1.2 too but seems like this API is new for TLS 1.3/